test/e2e/csv_e2e_test.go

```diff
@@ -4433,88 +4427,83 @@ var csvAnyChecker = buildCSVConditionChecker(operatorsv1alpha1.CSVPhasePending,
 var csvCopiedChecker = buildCSVReasonChecker(operatorsv1alpha1.CSVReasonCopied)
 
 func fetchCSV(c versioned.Interface, name, namespace string, checker csvConditionChecker) (*operatorsv1alpha1.ClusterServiceVersion, error) {
-	var fetchedCSV *operatorsv1alpha1.ClusterServiceVersion
+	var fetched *operatorsv1alpha1.ClusterServiceVersion
 	var err error
 
 	ctx.Ctx().Logf("waiting for CSV %s/%s to reach condition", namespace, name)
-	Eventually(func() (bool, error) {
-		fetchedCSV, err = c.OperatorsV1alpha1().ClusterServiceVersions(namespace).Get(context.TODO(), name, metav1.GetOptions{})
+	Eventually(func() bool {
+		fetched, err = c.OperatorsV1alpha1().ClusterServiceVersions(namespace).Get(context.TODO(), name, metav1.GetOptions{})
 		if err != nil {
-			return false, err
+			ctx.Ctx().Logf("error getting csv %s: %v", name, err)
+			return false
 		}
-		return checker(fetchedCSV), nil
+		ctx.Ctx().Logf("%s - %s (%s): %s", name, fetched.Status.Phase, fetched.Status.Reason, fetched.Status.Message)
```

Comment:
This kind of log message will spam on every iteration, please only log on edge transitions.

Reply:
1) WIP
2) Many of these logs ARE currently on every iteration
3) I had tried on transitions (which I agree is better), and it wasn't working.

I'll clean this up when I'm ready to commit.